src/ristretto/constants.rs

```diff
@@ -62,17 +62,26 @@ lazy_static! {
         }
         arr
     };
+
+    /// Precomputation tables for the points
+    pub static ref RISTRETTO_NUMS_TABLES: Vec<RistrettoBasepointTable> = {
+        let mut arr = Vec::<RistrettoBasepointTable>::new();
```

Comment:
A [lazy static](https://docs.rs/lazy_static/latest/lazy_static/) is perhaps better here. This code will always be invoked whenever the tari_crypto crate is used. Since there is computation and a heap allocation, I think doing this as needed/lazily makes sense.

```suggestion
        let mut arr = Vec::<RistrettoBasepointTable>::with_capacity(10);
```